proto/checkpoint/v1beta/query.proto

```diff
@@ -16,101 +16,108 @@ option (gogoproto.unmarshaler_all) = true;
-      option (google.api.http).get = "/heimdall/checkpoint/v1beta1/genesisstate/checkpoints";
+    // CheckpointList queries chepoing list .
+    rpc CheckpointList(QueryCheckpointListRequest)
+        returns (QueryCheckpointListResponse) {
+        option (google.api.http).get =
+            "/heimdall/checkpoint/v1beta1/genesisstate/checkpoints";
     }
 
-     // NextCheckpoint queries the next checkpoint.
-    rpc NextCheckpoint(QueryNextCheckpointRequest) returns (QueryNextCheckpointResponse) {
-      option (google.api.http).get = "/heimdall/checkpoint/v1beta1/genesisstate/checkpoints";
-    }   
-
+    // NextCheckpoint queries the next checkpoint.
+    rpc NextCheckpoint(QueryNextCheckpointRequest)
+        returns (QueryNextCheckpointResponse) {
+        option (google.api.http).get =
+            "/heimdall/checkpoint/v1beta1/genesisstate/checkpoints";
+    }
 }
 
 // QueryParamsRequest is request type for the Query/Params RPC method.
 message QueryParamsRequest {}
 
 // QueryParamsResponse is response type for the Query/Params RPC method.
 message QueryParamsResponse {
-  // params holds all the parameters of this module.
-  heimdall.checkpoint.v1beta1.Params params = 1 [(gogoproto.nullable) = false];
+    // params holds all the parameters of this module.
+    heimdall.checkpoint.v1beta1.Params params = 1
+        [(gogoproto.nullable) = false];
 }
 
 message QueryAckCountRequest {}
 
 message QueryAckCountResponse {
-  uint64 ack_count = 1;
+    uint64 ack_count = 1;
 }
 
 message QueryCheckpointRequest {
-  uint64 number = 1;
+    uint64 number = 1;
 }
 
 message QueryCheckpointResponse {
-  uint64 ack_count = 1;
-  heimdall.types.Checkpoint checkpoint = 2;
+    uint64                    ack_count  = 1;
+    heimdall.types.Checkpoint checkpoint = 2;
 }
 
 message QueryCheckpointBufferRequest {}
 message QueryCheckpointBufferResponse {
-  heimdall.types.Checkpoint checkpoint_buffer = 1;
+    heimdall.types.Checkpoint checkpoint_buffer = 1;
 }
 
 message QueryLastNoAckRequest {}
 message QueryLastNoAckResponse {
-  uint64   last_no_ack = 1;
+    uint64 last_no_ack = 1;
 }
 
 message QueryCheckpointListRequest {
-  heimdall.types.QueryPaginationParams pagination = 1;
+    heimdall.types.QueryPaginationParams pagination = 1;
 }
 
 message QueryCheckpointListResponse {
-  repeated heimdall.types.Checkpoint checkpoint_list = 1;
+    repeated heimdall.types.Checkpoint checkpoint_list = 1;
 }
 
 message QueryNextCheckpointRequest {
-  option (gogoproto.equal)           = false;
-  option (gogoproto.goproto_getters) = false;
+    option (gogoproto.equal)           = false;
+    option (gogoproto.goproto_getters) = false;
 
-  string bor_chainID = 1;
+    string bor_chainID = 1;
 }
 message QueryNextCheckpointResponse {
-  heimdall.checkpoint.v1beta1.MsgCheckpoint next_checkpoint = 1;
+    heimdall.checkpoint.v1beta1.MsgCheckpoint next_checkpoint = 1;
```

Comment:
We should add gogoproto annotations for multi-word keys. ref https://github.com/cosmos/cosmos-sdk/blob/v0.40.1/proto/cosmos/auth/v1beta1/auth.proto#L45

Reply:
I don't understand. Can you help here? cc @murthyvitwit

Reply:
We just need to change it to something like:
```go
heimdall.checkpoint.v1beta1.MsgCheckpoint next_checkpoint = 1 [(gogoproto.moretags) = "yaml:\"next_checkpoint\""];
```

Reply:
Sure, I will add them.

Reply:
Addressed in another PR (Gov)